script/entrypoint.sh

```diff
@@ -64,22 +64,23 @@ if [ "$AIRFLOW__CORE__EXECUTOR" != "SequentialExecutor" ]; then
   wait_for_port "Postgres" "$POSTGRES_HOST" "$POSTGRES_PORT"
 fi
 
+
 if [ "$AIRFLOW__CORE__EXECUTOR" = "CeleryExecutor" ]; then
   AIRFLOW__CELERY__BROKER_URL="redis://$REDIS_PREFIX$REDIS_HOST:$REDIS_PORT/1"
   wait_for_port "Redis" "$REDIS_HOST" "$REDIS_PORT"
 fi
 
 case "$1" in
   webserver)
-    airflow initdb
+    airflow db init
```

Comment:
@fathom-chanmin I saw you are using airflow upgrade, why? 

https://github.com/medicode/docker-airflow/blob/master/airflow_bridge/entrypoint.sh#L74